pages/common/pixiecore.md

```diff
@@ -0,0 +1,8 @@
+# pixiecore
+
+> Pixiecore is a tool to manage network booting of machines.
+> More information: <https://github.com/google/netboot/tree/master/pixiecore>.
```

Comment:
Im not too sure about this URL

Reply:
Surprising though it is, it does appear to be the official home for Pixiecore.

Reply:
It seems to automatically direct here instead.

https://github.com/danderson/netboot/tree/master/pixiecore

Reply:
Huh… yes… it's the address I thought I copy-pasted. Let me fix.